electron/protocol.ts

```diff
@@ -19,9 +19,14 @@ async function command_launch(
   window: BrowserWindow
 ) {
   const appName = args.appName ?? ''
-  const runner: Runner = (args.runner as Runner) ?? 'legendary'
+  let runner: Runner = (args.runner as Runner) ?? 'legendary'
 
-  const game = Game.get(appName, runner)
+  let game = Game.get(appName, runner)
+  // Fix for old protocol (which doesn't specify 'runner')
+  if (!game.getGameInfo()) {
+    runner = 'gog'
+    game = Game.get(appName, runner)
+  }
```

Comment:
The old code used this and it works:
```typescript
const game = Game.get(arg, 'legendary') || Game.get(arg, 'gog')
```

Reply:
`Game.get(arg, 'legendary')` with a non-existant AppName does not return a falsy value. This code does not work anymore

`console.log(game)` after passing a GOG AppName and `legendary`:
```
LegendaryGame {
  appName: '1293297882',
  window: BrowserWindow {
    setBounds: [Function (anonymous)],
    _events: [Object: null prototype] {
      blur: [Function (anonymous)],
      focus: [Function (anonymous)],
      show: [Function: visibilityChanged],
      hide: [Function: visibilityChanged],
      minimize: [Function: visibilityChanged],
      maximize: [Function: visibilityChanged],
      restore: [Function: visibilityChanged],
      close: [AsyncFunction (anonymous)]
    },
    _eventsCount: 8,
    devToolsWebContents: [Getter]
  },
  currentDownloadSize: 0
}
```

Reply:
Ok i see.
```
 const game =
      Game.get(arg, 'legendary').getGameInfo() ||
      Game.get(arg, 'gog').getGameInfo()
```